theia-example-updater/src/electron-browser/theia-updater-frontend-module.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import '../../src/electron-browser/style/index.css';
```

Comment:
```suggestion
import './style/index.css';
```
Can't the path be simplified to this?

Reply:
`Module not found: Error: Can't resolve './style/index.css' in '/project/theia-example-updater/lib/electron-browser'`
Not sure why though. In the Theia-Code they also import it with the longer path: https://github.com/eclipse-theia/theia/blob/38ca484cab170b92ff6a6f4ff6efb7e2c03b3e80/packages/core/src/browser/frontend-application-module.ts#L17
So I guess this should be fine